src/test/suite/utils/uiUtils.test.ts

```diff
@@ -94,4 +102,62 @@ suite('UIUtils Test Suite', () => {
         assert.equal(exceptionCount, 1);
         assert.equal(quickPickSpy.dispose.called, true); // ensure it was disposed of after item selected
     });
+
+    test('Static resources dir: workspace does not exist', async () => {
```

Comment:
You can remove these tests from this file, right? Looks like you (rightly) moved them into `workspaceUtils.test.ts` below.

Reply:
So strange because I removed them... Checked now and saw that `git push` hadn't gone through because I didn't add `-f` to it when the removal of those tests were done in ammend. 😞 . So much for haste makes waste.